src/solvers/dgsem_t8code/dg.jl

```diff
@@ -0,0 +1,30 @@
+@muladd begin
```

Comment:
```suggestion
@muladd begin
#! format: noindent

```
Please run the formatter again after this and add the comment on `@muladd` that we use elsewhere

Reply:
Done.